core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -84,7 +86,7 @@ public boolean exists(String path) throws IOException {
   @Override
   public Pair<AccessControlList, DefaultAccessControlList> getAclPair(String path)
       throws IOException {
-    return new Pair<>(null, null);
+    return EMPTY_ACL;
```

Comment:
Save obj creation by singleton. I don't find any attempt to update the return of this. But if we are concerned, we can find a way to make this one immutable.